src/components/Layouts/Layouts.stories.js

```diff
@@ -0,0 +1,208 @@
+/**
+ * @file Layout stories.
+ * @copyright IBM Security 2020
+ */
+
+import { boolean } from '@storybook/addon-knobs';
+import { storiesOf } from '@storybook/react';
+
+import React from 'react';
+import { Grid, Row, Column } from 'carbon-components-react';
+import ArrowRight20 from '@carbon/icons-react/lib/arrow--right/20';
+
+import {
+  Accordion,
+  AccordionItem,
+  Button,
+  Tab,
+  Tabs,
+  TypeLayout,
+  TypeLayoutBody,
+  TypeLayoutRow,
+  TypeLayoutCell,
+} from '../..';
+
+import { disableCentered, patterns } from '../../../.storybook';
+
+const toggleDebugging = () => boolean('Toggle PADDING debugging 🕵️‍♀️', false);
+
+const showAccordion = () => boolean('Show accordion', true);
+const showButtonGroup = () => boolean('Show button group', true);
+const showCardGroup = () => boolean('Show card group', true);
+const showTabGroup = () => boolean('Show tab group', true);
+
+const showFirstRow = showCardGroup() || showButtonGroup();
+const showSecondRow = showTabGroup() || showAccordion();
+
+disableCentered(storiesOf(patterns('Layouts'), module)).add(
+  'details page layout',
+  () => (
+    <main className={toggleDebugging() && 'security--debug--padding'}>
+      <h1 className="bx--type-productive-heading-04">productive-heading-04</h1>
+      <Grid condensed>
+        {showFirstRow && (
+          <Row>
+            {showCardGroup() && (
+              <Column md={3} lg={6}>
+                <h2 className="bx--type-productive-heading-01 security--padding-top--layout-01 security--padding-bottom--layout-03 security--margin-top--layout-00 security--margin-bottom--layout-00">
```

Comment:
This makes sense as we discussed, but seeing it in written out in code, it starts to become a fairly lengthy step for consumers? 🙀

If the 'spacing' for titles is always going to be this combination, it might be beneficial to roll these up into a single class for consumers to apply (with the exception of `bx--type-productive-heading-01`)? This is more a question to consider for the stories we're picking up this sprint, and I don't think we'll have the right answer until we work through them!

Reply:
@SimonFinney Yes they can be lengthy, but I would caution against "rolling them up" because it would get us back to an earlier iteration where we were over-abstracting too much. We can't, and shouldn't, always assume a certain combination of top + bottom padding here. There could be a number of iterations -- and how would the naming convention work for those?

This is lengthy but it is readable, understandable, customizable. 👍 

Now there IS an opportunity to create top + bottom and even "block" padding classes for *the same padding value* vertically or all around but that wouldn't change what you see here.

Reply:
We need to continue to be mindful of the pros and cons of abstraction - the previous iteration you're mentioning was one avenue of exploration and learning, and the outcomes of that, good and bad, should not necessarily be disregarded outright as it helped better tease out and inform the design concept with @cameroncalder . A combination of techniques should still be considered to inform ease-of-implementation by our consumers.

As you've mentioned, we've learned that it might not make sense to abstract out grid columns, because that's inflexible, but I don't believe that means we should potentially irritate all of our consumers to apply a string of classes à la `security--padding-top--layout-01 security--padding-bottom--layout-03 security--margin-top--layout-00 security--margin-bottom--layout-00 security--etc` when perhaps all they want is one thing that does it for them.

As I've said in the past, abstraction is a delicate balance and used everywhere - I encourage you to look at examples of where this is useful (see Carbon's `Grid` component), and also to wear the shoes of our consumers to understand how it can be beneficial, especially as we review the specifications for the layout modules and discover that recurring patterns do exist and there are implementation details that they should not care about.